cmd/gb-vendor/vendor/repo.go

```diff
@@ -163,7 +163,8 @@ func probeGitUrl(schemes []string, host, path string, insecure bool) (string, er
 		switch scheme {
 		case "https":
 			url := scheme + "://" + host + "/" + path
-			if _, err := run("git", "ls-remote", "--exit-code", url, "HEAD"); err == nil {
```

Comment:
Yes, that's probably fine. It should return error on non-existent repositories which is what's important, I guess?